test/clj/cider/nrepl/middleware/stacktrace_test.clj

```diff
@@ -122,14 +132,18 @@
             :path "/foo/bar/baz.clj"
             :line 1
             :column 42}
-           (extract-location {:class "clojure.lang.Compiler$CompilerException"
-                              :message "java.lang.RuntimeException: Unable to resolve symbol: foo in this context, compiling:(/foo/bar/baz.clj:1:42)"})))
+           (select-keys
```

Comment:
Won't it be better if you add `:full-file` to the result (even if it's stubbed)?

Reply:
Can do. I was being lazy.